webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java

```diff
@@ -29,5 +37,39 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         String firebaseResponse = Firebase.sendGetRequest("https://summer20-sps-47.firebaseio.com/orders.json?orderBy=%22userEmailRoomId%22&equalTo=%22" + userEmail + "_" + roomId + "%22");
         response.setStatus(200);
         response.getWriter().print(firebaseResponse);
+    }     
+    
+    @Override
+    public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        BufferedReader reader = request.getReader();
+        String orderId = reader.readLine();
+        orderId = orderId.split("=")[1];
+        String orderData = Firebase.sendGetRequest("https://summer20-sps-47.firebaseio.com/orders/" + orderId + ".json");
```

Comment:
I thought we established that we dont do this anymore for getting order?

Reply:
I've changed it!